docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/generic.md

```diff
@@ -90,28 +124,32 @@ spec:
           whenExpression: "request.header("X-Event-Type") == 'push'"
 ```
 
-### Defining TargetSelectionCriteria
+:::note
+This is can be left empty if the action should run unconditionally.
+:::
+
+#### Defining targetSelectionCriteria
 
-`TargetSelectionCriteria` is used to designate resources that an action needs
-to be performed on. There are three ways to define `TargetSelectionCriteria`:
+`targetSelectionCriteria` is used to select resources that an action needs
+to be performed on. There are three ways to define `targetSelectionCriteria`:
 
 1. [By Name](#by-name)
 2. [By Labels](#by-labels)
 3. [By Values in an Index](#by-values-in-an-index)
 
 All methods support both static values and 
-[expression-derived values](#expression-functions).
+[expression-derived values](#expression-reference).
 
 :::note
 Using more than one of the above results in a criteria set that is the logical
 **AND** of all defined criteria.
```

Comment:
This feels like more technically and grammatically correct terminology, but we should keep the more colloquial "AND" bit parenthetically because it's definitely a valuable bit of clarification.

```suggestion
Using more than one of the above selects resources at the logical intersection
of the criteria. (i.e. Criteria are "ANDed.")
```